app/app.js

```diff
@@ -117,11 +118,13 @@ apiRouter.use('/v1', v1Router);
 // Root level Router
 app.use(/(\/api)?/, apiRouter);
 
-// Handle 500
+// Handle ValidationError & 500
 // eslint-disable-next-line no-unused-vars
 app.use((err, _req, res, _next) => {
   if (err instanceof Problem) {
     err.send(res);
+  } else if (err instanceof ValidationError) {
+    return res.status(err.statusCode).json(err);
```

Comment:
it may be a good idea to log validation errors. it has value from a security perspective to monitor brute force attacks.
something from the STRA checklist.

Reply:
Not necessarily. Validation errors arise from external entity network traffic that contains parameters or queries that do not match our specifications. If we do decide to log these kinds of errors, they would need to be flagged at either the verbose or the lower debug level. Most enterprise level applications do not log anything above and beyond the http level for most situations. You can catch brute force attacks with just http request layer stuff already without any need for more explicit logging.